flatsurf/geometry/gl2r_orbit_closure.py

```diff
@@ -755,20 +755,20 @@ def lift(self, v):
 
             sage: T = polygons.triangle(3,4,13)
             sage: S = similarity_surfaces.billiard(T)
-            sage: S = S.minimal_cover("translation").erase_marked_points() # optional: pyflatsurf
-            sage: O = GL2ROrbitClosure(S)  # optional: pyflatsurf
-            sage: for d in O.decompositions(4, 20):  # optional: pyflatsurf
+            sage: S = S.minimal_cover("translation").erase_marked_points() # long time (3s), optional: pyflatsurf
+            sage: O = GL2ROrbitClosure(S)  # long time (above), optional: pyflatsurf
+            sage: for d in O.decompositions(4, 20):  # long time (2s), optional: pyflatsurf
             ....:     O.update_tangent_space_from_flow_decomposition(d)
             ....:     if O.dimension() == 4:
             ....:         break
-            sage: d1,d2,d3,d4 = [O.lift(b) for b in O.tangent_space_basis()]  # optional: pyflatsurf
-            sage: dreal = d1/132 + d2/227 + d3/280 - d4/201  # optional: pyflatsurf
-            sage: dimag = d1/141 - d2/233 + d4/230 + d4/250  # optional: pyflatsurf
-            sage: d = [O.V2((x,y)).vector for x,y in zip(dreal,dimag)]  # optional: pyflatsurf
-            sage: S2 = O._surface + d  # optional: pyflatsurf
-
-            sage: O2 = GL2ROrbitClosure(S2.surface())  # optional: pyflatsurf
-            sage: for d in O2.decompositions(4, 20):  # optional: pyflatsurf
+            sage: d1,d2,d3,d4 = [O.lift(b) for b in O.tangent_space_basis()]  # long time (above), optional: pyflatsurf
+            sage: dreal = d1/132 + d2/227 + d3/280 - d4/201  # long time (above), optional: pyflatsurf
+            sage: dimag = d1/141 - d2/233 + d4/230 + d4/250  # long time (above), optional: pyflatsurf
+            sage: d = [O.V2((x,y)).vector for x,y in zip(dreal,dimag)]  # long time (above), optional: pyflatsurf
+            sage: S2 = O._surface + d  # long time (6s), optional: pyflatsurf
+
+            sage: O2 = GL2ROrbitClosure(S2.surface())  # long time (above), optional: pyflatsurf
+            sage: for d in O2.decompositions(1, 20):  # long time (25s), optional: pyflatsurf
```

Comment:
`25s` looks a bit excessive. Too many iterations? (I was not able to get the loop finished on my computer).

Reply:
This is now #124.